File: java-files/17.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new FileReader("simple_example.txt"));
        String line;
        while ((line = reader.readLine())!= null) {
            System.out.println(line);
        }
        reader.close();
    }
}

File: cpp-test-files/17.cpp
//file
#include <iostream>
#include <fstream>
#include <string>

int main() {
    std::ifstream inFile("simple_example.txt");
    std::string line;
    while (std::getline(inFile, line)) {
        std::cout << line << std::endl;
    }
    inFile.close();
    return 0;
}